pysatNASA/instruments/timed_guvi.py

```diff
@@ -38,121 +60,86 @@
 
 import datetime as dt
 import functools
-import numpy as np
-import warnings
-import xarray as xr
 
-import pysat
 from pysat.instruments.methods import general as mm_gen
-from pysat import logger
 
 from pysatNASA.instruments.methods import cdaweb as cdw
+from pysatNASA.instruments.methods import general as mm_nasa
+from pysatNASA.instruments.methods import jhuapl
+from pysatNASA.instruments.methods import timed as mm_timed
 
 # ----------------------------------------------------------------------------
 # Instrument attributes
 
 platform = 'timed'
 name = 'guvi'
-tags = {'sdr-imaging': 'Level 1C imaging data',
+tags = {'edr-aur': 'Level 2 Auroral disk imaging mode',
+        'sdr-imaging': 'Level 1C imaging data',
         'sdr-spectrograph': 'Level 1C spectrograph data'}
-inst_ids = {'high_res': list(tags.keys()),
-            'low_res': list(tags.keys())}
+inst_ids = {'': ['edr-aur'],
+            'high_res': ['sdr-imaging', 'sdr-spectrograph'],
+            'low_res': ['sdr-imaging', 'sdr-spectrograph']}
 
 pandas_format = False
+multi_file_day = True
 
 # ----------------------------------------------------------------------------
 # Instrument test attributes
 
-_test_dates = {
-    'high_res': {tag: dt.datetime(2005, 6, 28) for tag in tags.keys()},
-    'low_res': {tag: dt.datetime(2005, 6, 28) for tag in tags.keys()}}
+_test_dates = {iid: {tag: dt.datetime(2005, 6, 28) for tag in inst_ids[iid]}
+               for iid in inst_ids.keys()}
+_test_load_opt = {iid: {tag: [{'combine_times': True}, {'combine_times': False}]
+                        for tag in inst_ids[iid]} for iid in ['high_res',
+                                                              'low_res']}
```

Comment:
```suggestion
_test_load_opt = {iid: {tag: {'combine_times': True}
                        for tag in inst_ids[iid]} for iid in ['high_res',
                                                              'low_res']}
```

There seems to be a bug on the pysat side.  It's doubling up on combine_times=False.  In general, I've only put the non-default option up into `_test_load_opt`.  This should fix the coverage issue.